.github/workflows/quarkus-snapshot.yaml

```diff
@@ -16,7 +16,7 @@ env:
   # Repo specific setting #
   #########################
 
-  ECOSYSTEM_CI_REPO_PATH: quarkus-pooled-jms
+  ECOSYSTEM_CI_REPO_PATH: quarkiverse-pooled-jms
```

Comment:
FWIW, this is fixed in the new Quarkiverse template that is coming soon with 2.10.1.Final.

Reply:
And https://github.com/quarkiverse/quarkus-pooled-jms/pull/2/files could be fixed?

Reply:
Yes it's also fixed already :).